lib/index.d.ts

```diff
@@ -26,16 +28,9 @@ export enum SIGNATURE_TYPES {
     CANONICAL_TEXT = 1
 }
 
-// type defined in OpenPGP is not complete
-export interface OpenPGPKey extends key.Key {
-    users?: { userId?: { userid?: string } }[];
-    getExpirationTime( // override upstream definition, since it does not properly declare `Infinity` as return type
-        capabilities?: "encrypt" | "sign" | "encrypt_sign", keyId?: type.keyid.Keyid, userId?: object
```

Comment:
verify `capabilities` is correctly replaced here and in the apps (`getExpirationTime` has changed behaviour)